src/Network/Receive/kRO/Sakexe_0.pm

```diff
@@ -3638,5 +3640,34 @@ sub progress_bar_unit {
 	debug "Displays progress bar (GID: $args-{GID} time: $args-{time})\n";	
 }
 
+sub inventory_expansion_info {
+	my($self, $args) = @_;
+	#sd->inventorySize - FIXED_INVENTORY_SIZE; 
+	#hardcode inventorysize ? [sctnightcore]
```

Comment:
My idea : 
my $size = $char->{'weight_max'} - $args->{expansionSize};
message TF("You have more storage space %s",$size),"info";